src/handlers/get-random-hex.handler.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { getRandomHexHandler } from './get-random-hex.handler'
+
+describe(`getRandomHexHandler`, () => {
+  it(`should expose a function`, () => {
+    expect(getRandomHexHandler).toBeDefined()
+  })
+
+  const DEFAULT_HEX_LENGTH = 6
+  interface Test {
+    hexLength?: number
+    wantLength: number
+  }
+
+  const tests: Test[] = [
+    {
+      hexLength: undefined,
+      wantLength: DEFAULT_HEX_LENGTH,
+    },
+    {
+      hexLength: 1,
+      wantLength: 1,
+    },
+    {
+      hexLength: 2,
+      wantLength: 2,
+    },
+    {
+      hexLength: 6,
+      wantLength: 6,
+    },
+    {
+      hexLength: 10,
+      wantLength: 10,
+    },
+  ]
+
+  it(`getRandomHexHandler should return the expected output with correct length`, () => {
```

Comment:
This chunk of code should follow certain standard of writing test of typescript unit. Or use a auto generator from a proper source (facebook sized company or similar tech lead companies)

Reply:
As todo was written inside of the code, I will say it is okay to merge.